README.md

```diff
@@ -75,5 +75,7 @@ Run fifth scenario using command:
 ```bash
 docker-compose up scenario_5
 ```
+# Known issues
```

Comment:
We should remove this section as soon as we update the private node with the DDC submodule as well.

Reply:
Absolutely right.

Reply:
Right!